whisper-finetune/inference_endpoint/inference.py

```diff
@@ -0,0 +1,39 @@
+import urllib.request
+import json
+import librosa
+
+# Request data goes here
+# The example below assumes JSON formatting which may be updated
+# depending on the format your endpoint expects.
+# More information can be found here:
+# https://docs.microsoft.com/azure/machine-learning/how-to-deploy-advanced-entry-script
+audio = librosa.load("../common_voice_hi_23795238.mp3")[0]
+audio = audio.tolist()
+data = {"audio": audio}
+
+body = str.encode(json.dumps(data))
+
+# TODO: Replace this with the your REST endpoint link
```

Comment:
may we add TODO(alias/username/name) so we can track this in the future?

Reply:
Adjusted to hide credentials the way other demos in this repo hide them.